sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/GeneratorExpressionSuite.scala

```diff
@@ -68,4 +68,23 @@ class GeneratorExpressionSuite extends SparkFunSuite with ExpressionEvalHelper {
       PosExplode(CreateArray(str_array.map(Literal(_)))),
       str_correct_answer.map(InternalRow.fromSeq(_)))
   }
+
+  test("inline") {
+    val correct_answer = Seq(
+      Seq(0, UTF8String.fromString("a")),
+      Seq(1, UTF8String.fromString("b")),
+      Seq(2, UTF8String.fromString("c")))
+
+    checkTuple(
+      Inline(CreateArray(Seq.empty)),
```

Comment:
we can use `Literal.create(Array(), ArrayType(...))` to create a empty array with expected type.